stx-genesis/chainstate-test.txt

```diff
@@ -1,25 +1,25 @@
 -----BEGIN NAMESPACES-----
-namespace_id,address,buckets,base,coeff,nonalpha_discount,no_vowel_discount,lifetime
-blockstack,SP355AMV5R2A5C7VQCF4YPPAS05W93HTB68574W7S,7;6;5;4;3;2;1;1;1;1;1;1;1;1;1;1,4,250,4,4,0
-graphite,SP3X2W6GTGVVG9RW78F1MT7M9AQFS2HWV2JETXPG2,6;5;4;3;2;1;0;0;0;0;0;0;0;0;0;0,4,250,4,4,52595
-helloworld,SP3TJ4WRTXW0YFWF1TWKWCG7E6Z6MZDTWAVB6SW5V,15;14;13;12;11;10;9;8;7;6;5;4;3;2;1;0,2,4,2,5,0
-id,SP364R1Y0XYC8PYXEK0PE3V752RAJB2GNWF28WP90,6;5;4;3;2;1;0;0;0;0;0;0;0;0;0;0,4,250,4,4,52595
-podcast,SPHYHD2ZJ11HWRDKVZ160QX7FD7PCVNT2N9CT2H6,1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1,40,250,1,1,0
+namespace_id,address,reveal_block,ready_block,buckets,base,coeff,nonalpha_discount,no_vowel_discount,lifetime
+blockstack,SP355AMV5R2A5C7VQCF4YPPAS05W93HTB68574W7S,524363,524393,7;6;5;4;3;2;1;1;1;1;1;1;1;1;1;1,4,250,4,4,0
+graphite,SP3X2W6GTGVVG9RW78F1MT7M9AQFS2HWV2JETXPG2,522128,524057,6;5;4;3;2;1;0;0;0;0;0;0;0;0;0;0,4,250,4,4,52595
+helloworld,SP3TJ4WRTXW0YFWF1TWKWCG7E6Z6MZDTWAVB6SW5V,490390,497855,15;14;13;12;11;10;9;8;7;6;5;4;3;2;1;0,2,4,2,5,0
+id,SP364R1Y0XYC8PYXEK0PE3V752RAJB2GNWF28WP90,373608,384057,6;5;4;3;2;1;0;0;0;0;0;0;0;0;0;0,4,250,4,4,52595
+podcast,SPHYHD2ZJ11HWRDKVZ160QX7FD7PCVNT2N9CT2H6,498126,506980,1;1;1;1;1;1;1;1;1;1;1;1;1;1;1;1,40,250,1,1,0
 -----END NAMESPACES-----
 -----BEGIN NAMES-----
-name,address,zonefile_hash
-0.id,SP1HPCXTGV31W5659M3WTBEFP5AN55HV4B1Q9T31F,c5f5b4197c94d4561f6332160f624d728b0da8bf
-1.id,SP1FRDVA4JAZ4TC93XZC850QAY6R4MTHN10XZJ035,8961c32ad98df87af12b186c3d0be6ca22105936
-10.id,SP12TFSBQFZWVVEAWJTVRW3K8V00R0S4895ABXRBR,8ae7173b9a2f96521ba077bbcbe9b56b1cf2449f
-10x.id,SP3N5TVVEHGA8B6G3JXWBP476161E32654D9P9A3Y,3d0576cd134fa1ff8d54d8e2ef5da97f5cae36f2
-111111111.id,SP15XBGYRVMKF1TWPXE6A3M0T2A87VYSVF9VFSZ1A,01a8c60c852b75f9c6417491ac89275fb9de774c
-123.id,SP2PEZ0W4PFGYV6ZZ2DWDVNAVVTQPJ9KX8EK4VRVT,ae34427289a8865028eb2d475dafb085060d5625
-zinai.id,SP19QQ0RZ2F3CKJ2G2FT4EA905MKV89FY9K5XB5M3,c7b0bbeb92c2575a3e4a82632d8ec4b7fc86cc19
-zlh.id,SP1E311NW655FXQMQQKEKQKZZJFAA58AVP3GBTAZT,da6a954fdf90c890966cde5ae808ca6c96082428
-zone117x.id,SP3R9402R2XB0WMG0BTEMZ5Q67QFJ7Y8YVJRQCJZN,578ed433dabeb24952c892d5f3fc0cd7b9844bb8
-zumminer_crux.id,SP3DGYX27MM12SSPVGK2NMTR4836DT9VWYJV844BK,fa602059cb01dbe8d1885d8fc2a7c6b265c57e64
-zumminer_dev_crux.id,SPSSW2FFJQ2AMR5KZASBY8XH3F5WXHJV0M22GJ9G,f9569aa25b6325b0f2e3a5f9edb552cd98260f9b
-zumrai.id,SP29EJ0SVM2TRZ3XGVTZPVTKF4SV1VMD8C0GA5SK5,853cd126478237bc7392e65091f7ffa5a1556a33
+name,address,registered_at,expire_block,zonefile_hash
```

Comment:
It doesn't look like we're using `registered_at` or `expire_block`, right?

Reply:
That's correct, but it is what the Stacks 1.0 node outputs.

Reply:
I'm pretty sure that those fields were removed from the dump.
Is it possible that we mixed up commits in that branch @zone117x ?

Reply:
Good catch -- yeah, I think that's what's happening -- an older `v1-migration` core node was used to digest the stacks-node backup data and open #2324, which also explains why things weren't re-mapped.